lib/rubocop/config_loader.rb

```diff
@@ -18,7 +18,7 @@ class ConfigLoader
     class << self
       include ConfigLoaderResolver
 
-      attr_accessor :debug, :auto_gen_config
+      attr_accessor :debug, :auto_gen_config, :ignore_parent_exclusion
```

Comment:
As with these two other boolean attributes, add a predicate alias `ignore_parent_exclusion?`.

Reply:
Spec is now in place.